### PR TITLE
Add default logger

### PR DESCRIPTION
**Pull request by dmtsui**

### CODE REVIEW

**Comment by lxe** [comment]
The logger API should be that of `[Console](https://nodejs.org/api/console.html)`, not `logtron`, since it's more widely used.

File: canduit.js [hunk 1]
```diff
@@ -14,21 +15,19 @@ function createCanduit (opts, cb) {
...
-    log: function silent () { }
-  };
+  var self = this;
+  self.client = opts.client || 'canduit';
+  self.logger = opts.logger || NullLogtron();
 
-  this.api = opts.api;
-  this.user = opts.user;
-  this.cert = opts.cert;
+  self.api = opts.api;
```
Comment (lxe) [inline canduit.js]: No need for the `self = this` unless you are creating closure scopes.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 node_modules
 coverage
+.idea
```
Comment (lxe) [inline .gitignore]: This is better suited for the global gitignore.

**Comment by dawsonbotsford** [comment]
Closing this due to inactivity. If the requested changes are made, it will be reconsidered @dmtsui

**Pull request closed without merging**